src/Analyzer/DurableFunctionsAnalyzer/DurableFunctionsAnalyzer/AnalyzerRegistration.cs.orig

```diff
@@ -0,0 +1,60 @@
+using DurableFunctionsAnalyzer.Analyzers;
```

Comment:
Can this file be removed? It looks like an artifact from merge conflict resolution.